python-sdk/docs/CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## 1.4.0
+
+### Feature:
+- Support for SFTP as file location [#1481](https://github.com/astronomer/astro-sdk/pull/1481)
+- Support for FTP as file location [#1482](https://github.com/astronomer/astro-sdk/pull/1482)
+- Add support for Azure Blob Storage [#1275](https://github.com/astronomer/astro-sdk/pull/1275), [#1542](https://github.com/astronomer/astro-sdk/pull/1542)
+- Add databricks support [#1352](https://github.com/astronomer/astro-sdk/pull/1352), [#1397](https://github.com/astronomer/astro-sdk/pull/1397), [#1452](https://github.com/astronomer/astro-sdk/pull/1452), [#1476](https://github.com/astronomer/astro-sdk/pull/1476), [#1480](https://github.com/astronomer/astro-sdk/pull/1480), [#1555](https://github.com/astronomer/astro-sdk/pull/1555)
+- Add source code facet for operators [#1537](https://github.com/astronomer/astro-sdk/pull/1537)
+- Use load_options param to pass pandasOptions in load_file [#1466](https://github.com/astronomer/astro-sdk/pull/1466)
```

Comment:
Example of this is needed for user

Reply:
WDYT about https://github.com/astronomer/astro-sdk/pull/1543#discussion_r1067262020

Reply:
https://github.com/astronomer/astro-sdk/releases/tag/1.3.0